.gitignore

```diff
@@ -0,0 +1,2 @@
+#Ignore node_modules
+node_modules/
```

Comment:
Remember to add your [test.md](https://github.com/Chimwemwe-127001/cr-hello-microverse/blob/276029dfa822d161e2d8a234043678f3d16b54bc/test.md) file here, as stated by the project requirements:
![image](https://user-images.githubusercontent.com/78335447/169365100-55da5ed8-2921-4a5a-be0d-e85fbddb0988.png)
This should be easy and quick :wink: